source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -151,6 +151,8 @@ We upgrade PTL/PTLSbox and Preview/Dev in-place, meaning we aren't building anot
 This environment is best to be done early in the morning, that Jenkins may be briefly unavailable on the relevant #cloud-native-announce slack channel.
 Subsequently, please verify the status of [Jenkins instances] for CFT (https://build.hmcts.net/) and SDS (https://sds-build.hmcts.net/) to ensure they are operational following the upgrades. 
 
+We upgrade PTL/PTLSbox and Preview/Dev environments by performing an in-place upgrade on the current live cluster. This means we aren't building another cluster or switching in the Application Gateway unless necessary. If one of these environments must instead be rebuilt and swapped over, please refer to the [Rebuilding AKS Clusters guide](https://hmcts.github.io/ops-runbooks/aks/rebuilding-aks-clusters.html#environment-specific).
+
 - Update Ptlsbox Jenkins to switch from 00 to 01 if you are upgrading 00 cluster
```

Comment:
Can you remove this please / link off to the rebuild section if needed. This change isn't clear if below it still says to switch clusters when doing an upgrade

Reply:
done